bin/wazo-service

```diff
@@ -108,17 +108,17 @@ start_monit() {
 
 close_sip_port() {
     echo "Closing port 5060."
-    iptables -n --list | grep -q '^DROP.*5060'
+    nft -n list table filter | grep -q "^\s*udp dport 5060 counter packets 0 bytes 0 drop"
     if [ $? -eq 1 ]; then
-        iptables -I INPUT 1 -p udp --dport 5060 -j DROP
+        nft insert rule ip filter INPUT udp dport 5060 counter drop
```

Comment:
The iptables doc is explicit about where the rule is added
> Insert one or more rules in the selected chain as the given rule number. So, if the rule number is 1, the rule or rules are inserted at the head of the chain. This is also the default if no rule number is specified. 

But I didn't find this kind of behavior in the man of nft. Can you confirm that if we don't specify the rule number, it happen before everyone?

Reply:
In nftables they seem to be appended. If you want to specify location it looks like you need to know the handle.
https://wiki.nftables.org/wiki-nftables/index.php/Simple_rule_management.
It is not specified in the official documentation, but it is in the [Archlinux doc](https://wiki.archlinux.org/title/nftables#:~:text=nftables%20is%20a%20netfilter%20project,%7Bip%2Cip6%7Dtables.).